DMTN-306.tex

```diff
@@ -1,79 +1,119 @@
+
+\subsection{From archive to processing facilities and back}
+\label{summit-to-data-facilities}
+
+Annual processing of the entire image dataset recorded since the beginning of the survey is carried out across three facilities: the US Data Facility, hosted at SLAC National Laboratory in California, USA\footnote{https://www.slac.stanford.edu}, the French Data Facility, hosted by the IN2P3 computing center (CC-IN2P3) in Lyon, France\footnote{https://cc.in2p3.fr}, and the UK Data Facility, operated by the LSST:UK consortium\footnote{https://www.lsst.ac.uk}.
+
+Raw image data is replicated from the US to the European facilities. Both the US and French data facilities store a complete copy of the raw image dataset. The UK facility receives the raw images corresponding to the spatial region assigned to it for processing. Data movement between these sites is facilitated by ESnet\footnote{https://es.net}, which handles transatlantic data transport; GEANT\footnote{https://geant.org}, which connects European sites; and the national research and education networks, JANET\footnote{https://www.jisc.ac.uk/janet} (UK) and RENATER\footnote{https://renater.fr} (France).
+
+The entire set of final data products, along with selected intermediate products from each campaign, is replicated from the facility where they are generated to the archive center. There, they are consolidated and incorporated into a new data release, which is delivered annually to the science collaborations for analysis \cite{10.1051_epjconf_202429501042}.
+
+The LSST Science Pipelines is the software developed by Rubin Observatory to process the survey data \cite{bosch-pipelines}. It includes advanced image processing algorithms and supporting middleware. A central component of this middleware is the Rubin Data Butler, an abstraction layer that mediates access to the data required by, or generated through, the pipelines \cite{2022SPIE12189E..11J}. The Data Butler retrieves data from persistent storage (using appropriate protocols and data formats) based on queries specified by scientifically relevant identifiers (rather than file paths), and delivers the data as in-memory Python objects to the pipelines. It also persists the in-memory objects generated by the science algorithms. Crucially, the Butler manages the location of all files within the data store, recording their locations and relationships in a relational database. Together, the file registry and the storage system where files are located constitute a \emph{repository}.
+
+Since the Rubin's data Butler is aware only of the files present at a single facility, files replicated between facilities need to be placed in the repository's data store at the location expected by the Butler. Upon reception, replicated files are ingested into the receiving facility's local Butler repository, making them available for the processing pipelines.
```

Comment:
```suggestion
Since a given Butler repository is aware only of the files present at a single facility, files replicated between facilities need to be placed in the repository's data store at the location expected by the Butler. Upon reception, replicated files are ingested into the receiving facility's local Butler repository, making them available for the processing pipelines.
```

Reply:
Implemented.